handlers/lookup.go

```diff
@@ -75,6 +80,21 @@ func (l *lookupHandler) handleMissingRoute(rw http.ResponseWriter, r *http.Reque
 	)
 }
 
+func (l *lookupHandler) handleUnavailableRoute(rw http.ResponseWriter, r *http.Request) {
+	rw.Header().Set("X-Cf-RouterError", "no_endpoints")
```

Comment:
The code here looks similar to the code from `handleMissingRoute`. Maybe we can extract that logic?

Reply:
Done in https://github.com/cloudfoundry/gorouter/pull/251/commits/5936addabfcf0061c4c5ebe61ddf1239389b0d19